docs/pages/about.mdx

```diff
@@ -0,0 +1,11 @@
+import { MarkdownPageTemplate } from '../components/MarkdownPageTemplate';
```

Comment:
Lets just make this `about.tsx` and remove all of the mdx related code.  This is consistent with how other static pages work at the moment.